src/pretix/plugins/banktransfer/urls.py

```diff
@@ -21,6 +29,15 @@
         views.EventJobDetailView.as_view(), name='import.job'),
     url(r'^control/event/(?P<organizer>[^/]+)/(?P<event>[^/]+)/banktransfer/action/',
         views.EventActionView.as_view(), name='import.action'),
+    url(r'^control/event/(?P<organizer>[^/]+)/(?P<event>[^/]+)/banktransfer/refunds/',
+        views.EventRefundExportListView.as_view(),
+        name='refunds.list'),
+    url(r'^control/event/(?P<organizer>[^/]+)/(?P<event>[^/]+)/banktransfer/export/(?P<id>\d+)/$',
+        views.EventDownloadRefundExportView.as_view(),
+        name='refunds.download'),
+    url(r'^control/event/(?P<organizer>[^/]+)/(?P<event>[^/]+)/banktransfer/sepa-export/(?P<id>\d+)/$',
+        views.EventSepaXMLExportView.as_view(),
+        name='refunds.sepa'),
 ]
 
```

Comment:
Can you add all these to the various array in ``test_permissions.py`` to make sure they are restricted correctly?